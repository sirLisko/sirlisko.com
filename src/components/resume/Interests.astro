<article class="section interests">
  <h1>Interests</h1>
  <p>
    Passionate about <a href="https://www.last.fm/user/sirlisko">music</a>,{" "}
    <a href="https://www.imdb.com/user/ur15844029/">cinema</a>,{" "}
    <a href="https://www.goodreads.com/sirLisko">books</a>, and, of course, <a
      href="https://pizzaclub.sirlisko.com/">pizza</a
    >.
  </p>
  <p>
    Enthusiastic about modern art, 8-bit graphics (like old-school video games),
    travel, languages. I'm also an active member of{" "}
    <a href="https://www.couchsurfing.com/people/sirlisko">
      CouchSurfing community
    </a>
     .
  </p>
</article>
