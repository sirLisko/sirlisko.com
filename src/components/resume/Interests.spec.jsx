import React from "react";
import renderer from "react-test-renderer";

import Interests from "./Interests.jsx";

describe("Interests Component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = renderer.create(<Interests />).toJSON();
  });

  it("should render properly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
