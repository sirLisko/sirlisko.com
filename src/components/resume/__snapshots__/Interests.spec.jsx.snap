// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Interests Component should render properly 1`] = `
<article
  className="section interests"
>
  <h1>
    Interests
  </h1>
  <p>
    I like very much 
    <a
      href="https://www.last.fm/user/sirlisko"
    >
      music
    </a>
    ,
     
    <a
      href="https://www.imdb.com/user/ur15844029/"
    >
      cinema
    </a>
    ,
     
    <a
      href="https://www.goodreads.com/sirLisko"
    >
      books
    </a>
    , and, last but not least, 
    <a
      href="https://pizzaclub.sirlisko.com/"
    >
      pizza
    </a>
    .
  </p>
  <p>
    I like Art, especially modern and 8-bit graphic (like old-school video games), travel, languages. Iâ€™m also an active member of
     
    <a
      href="https://www.couchsurfing.com/people/sirlisko"
    >
      CouchSurfing community
    </a>
    .
  </p>
</article>
`;
